_data/simple-icons.json

```
@@ -6588,6 +6588,11 @@
             "source": "https://mozilla.design/mozilla/",
             "guidelines": "https://mozilla.design/mozilla/"
         },
+        {
+            "title": "mpv",
+            "hex": "520053",
```

```suggestion
            "hex": "691F69",
```
They seem to be using different shades of purple on their website. I took the one used by the play button on the [SVG found here](https://github.com/mpv-player/mpv/blob/da400ed3a9b54408d1b5112855a7281380a1ef52/etc/mpv.svg)